tests/testthat/setup-workflow-ref.R

```diff
@@ -19,7 +19,13 @@ ABS_MODEL_DIR <- system.file("model", "nonmem", "basic",   package = "rbabylon")
 MOD_ID <- "1"
 MODEL_DIR <-   fs::path_rel(ABS_MODEL_DIR, getwd()) %>% as.character()
 MOD1_PATH <- file.path(MODEL_DIR, MOD_ID)
-MOD1 <- MOD1_PATH %>% read_model()
+MOD1 <- read_model(MOD1_PATH)
+# if we're on Metworx or Drone, run the summary
+if (!Sys.getenv("METWORX_VERSION") == "" || Sys.getenv("DRONE") == "true") {
```

Comment:
Consider abstracting this logic into two functions, `is_metworx()` and `is_drone()`, and then calling them here and in the test helper.